### PR TITLE
Fix wait on router deployment for ingress operator daemonset.

### PR DESCRIPTION
**Pull request by ramr**
Follow up based on @abhinavdahiya  comments on #1952

@abhinavdahiya @crawford   PTAL thx

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
consistency across all if conditionals would be better for readability.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -148,7 +150,7 @@ objects:
 
         i=0
         MAX_RETRIES=10
-        until oc wait deploy/router -n "${ROUTER_NAMESPACE}" --for condition=available --timeout=10m || [ $i -eq $MAX_RETRIES ]; do
+        until oc wait ${ROUTER_DEPLOYMENT} -n "${ROUTER_NAMESPACE}" --for condition=available --timeout=10m || [ $i -eq $MAX_RETRIES ]; do
```
Comment (abhinavdahiya) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: quotes are not required?
`"${ROUTER_NAMESPACE}"` has quotes...

Comment (ramr) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: well quotes would be required if there's spaces in the actual name of the object (its `router[-default]` currently) I can put those in.

**Comment by ramr** [comment]
@abhinavdahiya made the requested changes PTAL thx

**Pull request merged**